Tests/FluentMongoDriverTests/FluentMongoDriverTests.swift

```diff
@@ -63,7 +63,7 @@ final class FluentMongoDriverTests: XCTestCase {
             ?? "localhost"
         try! self.dbs.use(.mongo(connectionString: "mongodb://\(hostname):27017/vapor-database"), as: .mongo)
         // Drop existing tables.
-        try! (self.db as! MongoDatabaseRepresentable).raw.drop().wait()
+        try! (self.db as! MongoDatabaseRepresentable).mongoKitten.drop().wait()
```

Comment:
Looks like this change is failing on CI.

Reply:
Oh, I thought I committed the change back. I did a find & replace on this.